### ISSUE TITLE
short imports are not squeezed to one line

### ISSUE THREAD

**Issue by radkujawa** [psf/black#1782]
**To Reproduce** Steps to reproduce the behavior:

1. Take this file:
```
from a import (
    b,
    c,
)
```
2. run black on it
3. file does not change

**Expected behavior** I would expect this file to be formatted in the following way:
```
from a import b, c
```

**Environment**

- Version: master, 20.8b1
- OS and Python version: python 3.7.9, ubuntu 20
- black config file (pyproject.toml):
```
[tool.black]
line-length = 120
target-version = ['py37']
skip-string-normalization = true
```

**Does this bug also happen on master?** yes

**Remark** 
the following (without trailing comma after `c`) is formatted correclty:
```
from a import (
    b,
    c
)
```

**Comment by MarcoGorelli** [comment]
@radkujawa if you remove the trailing comma they'll be squeezed on one line

**Comment by hugovk** [comment]
This works as designed, it's called the "magic trailing comma":

* https://black.readthedocs.io/en/stable/the_black_code_style.html?#the-magic-trailing-comma

**Comment by ichard26** [comment]
Black used to remove the trailing comma if the expression fits in a single line, but this was changed by #826 and #1288. Now a trailing comma tells Black to always explode the expression. This change was made mostly for the cases where you _know_ a collection or whatever will grow in the future. Having it always exploded as one element per line reduces diff noise when adding elements. Before the "magic trailing comma" feature, you couldn't anticipate a collection's growth reliably since collections that fitted in one line were ruthlessly collapsed regardless of your intentions. One of Black's goals is reducing diff noise, so this was a good pragmatic change.

So no, this is not a bug, but an intended feature. The reason why you're filing this issue is probably since we say the following in the (outdated) style documentation:
> Unnecessary trailing commas are removed if an expression fits in one line. This makes it 1% more likely that your line won't exceed the allotted line length limit. Moreover, in this scenario, if you added another argument to your call, you'd probably fit it in the same line anyway. That doesn't make diffs any larger.

We missed that this paragraph became incorrect when the "magic trailing comma" feature was introduced. It was eventually fixed in commit 6b935a34d08c0b18d858e7da5093dc69a29c55f2, but that was after the stable documentation was released alongside `Black 20.8b1`. 

Anyway, [here's the documentation](https://github.com/psf/black/blob/master/docs/the_black_code_style.md#the-magic-trailing-comma) on the "magic trailing comma". Hopefully that helps and sorry for the possible confusion.

**Comment by radkujawa** [comment]
thanks for quick response!
